drivers/RawSerial.h

```diff
@@ -59,10 +60,15 @@ namespace mbed {
  * }
  * @endcode
  */
-class RawSerial: public SerialBase, private NonCopyable<RawSerial> {
+class
+    MBED_DEPRECATED_SINCE(
+        "mbed-os-5.15.0",
```

Comment:
Won't actually be 5.15, right?

Reply:
Indeed, it is too late for 5.15 now. So next one

Reply:
[Changed to 6.0.0](https://github.com/ARMmbed/mbed-os/compare/6f17d09bc079d4788ce1c087c7236a96d66f4dec..707656ffd12e37cbe324228bcf83e327a1ae0e10).